client-misk/src/main/kotlin/app/cash/backfila/client/misk/BackfilaManagementClient.kt

```diff
@@ -3,11 +3,11 @@ package app.cash.backfila.client.misk
 import okio.ByteString
 
 interface BackfilaManagementClient {
-  fun <B : Backfill<*, *, P>, P : Any> createAndStart(
+  fun <B : Backfill> createAndStart(
     backfillClass: Class<B>,
     dry_run: Boolean,
     num_threads: Int? = null,
-    parameters: P? = null,
+    parameters: Any? = null,
```

Comment:
maybe we can eventually bring this back by having a Parameters type on the Backfill interface. makes sense given the API has the parameters built in

Reply:
I guess that is the question. I do think that it can make sense to have parameters be part of every Backfill.

It means that certain Backfill types may have a base type of Parameters object. I do think that is ok. One drawback is that you won't be able to use some externally defined class directly. You would need something to wrap it. Which is probably not a huge deal TBH.

I also think that I want the test functions to continue to be able to take arbitrary objects.